container/secrets.go

```diff
@@ -81,5 +81,13 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	baseSecrets, err := readAll("/usr/share/rhel/secrets", "")
+	if err != nil {
+		return nil, err
```

Comment:
Could we add a little documentation to this error.

return nil, fmt.Errorf("Failed to read /usr/share/rhel/secrets: %s", err.Error())